src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_afs_commands.py

```diff
@@ -23,7 +23,7 @@
 
 
 class BackupTests(ScenarioTest, unittest.TestCase):
-    # @record_only()
+    @record_only()
```

Comment:
Why change those tests to `@record_only`?

Reply:
This is how we use it in other test files too(test_backup_commands and test_workload_commands). For test_afs_commands too we were using @record_only earlier. But in the last push someone changed it (link to that PR - https://github.com/Azure/azure-cli/commit/e0d7b53da27cd6c7242456cfd8cecc00fe5fe04a#diff-10e8e53f12159505449845faf74cfcb07928e075ab11aff5fbc0ed95d48bb520 ). 
I am not sure why it was done so i reverted back to how we used to have it.

Reply:
@PARADISSEEKR May I ask why `@record_only` was deleted for these tests in PR #17831?

Reply:
> @PARADISSEEKR May I ask why `@record_only` was deleted for these tests in PR #17831?

`test_afs_commands.py` can use `@XXXPreparer` to create test resources for `azdev test test_afs_commadns --live` like `test_backup_commands.py`, but `test_workload_commands` cannot，so I deleted `@record_only` in `test_afs_commands.py` and kept `@record_only` in `test_workload_commands`.

Reply:
Will using @record_only tag cause any issue in test_afs_commands? Do I need to remove it or it's fine.

Reply:
Commented record_only tag in test_afs_commands.py